cfgov/unprocessed/css/print.less

```diff
@@ -68,6 +71,31 @@
         border-bottom-width: 0;
     }
 
+    a.a-btn {
+        // These pseudo-classes are needed to guarantee specificity
+        &:link,
+        &:visited {
+            // Undo all button and stand-alone link stylings and add some generic styling
```

Comment:
Undo from the Design System styles? Maybe say something like `Override Design System button and stand-alone link stylings …`